presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -68,6 +68,7 @@
     public static final String LEGACY_ORDER_BY = "legacy_order_by";
     public static final String REORDER_WINDOWS = "reorder_windows";
     public static final String ITERATIVE_OPTIMIZER = "iterative_optimizer_enabled";
+    public static final String AGGREGATION_PUSHDOWN = "aggregation_pushdown_enabled";
```

Comment:
sure,  I was modeling the name after the iterative optimizer property above it.  But your example seems more relevant.